packages/slice-machine/src/modules/selectedCustomType/types.ts

```diff
@@ -8,6 +8,7 @@ export enum CustomTypeStatus {
   New = "NEW_CT",
```

Comment:
I'm wondering about these status,
so there's one for when it's new
one for when it's modified,
one for when it's synced.... with?
....like it could be synced with the file system or synced with the api?

Reply:
`synced` = synced with the api and `modified` = local is different from remote i believe!